ui/components/Header.js

```diff
@@ -1,194 +1,219 @@
+import AppBar from '@material-ui/core/AppBar';
+import Grid from '@material-ui/core/Grid';
+import Hidden from '@material-ui/core/Hidden';
+import IconButton from '@material-ui/core/IconButton';
+import MenuIcon from '@material-ui/icons/Menu';
+import Toolbar from '@material-ui/core/Toolbar';
+import Typography from '@material-ui/core/Typography';
+import { withStyles } from '@material-ui/core/styles';
+import { connect } from 'react-redux';
+import NoSsr from '@material-ui/core/NoSsr';
+import Link from 'next/link';
+import SettingsIcon from '@material-ui/icons/Settings';
+import DashboardIcon from '@material-ui/icons/Dashboard';
+import MesheryNotification from './MesheryNotification';
+import User from './User';
+// import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+// import { faHome } from '@fortawesome/free-solid-svg-icons';
+
+const lightColor = 'rgba(255, 255, 255, 0.7)';
+
+const styles = (theme) => ({
+  secondaryBar: {
+    zIndex: 0,
+  },
+  menuButton: {
+    marginLeft: -theme.spacing(1),
+  },
+  iconButtonAvatar: {
+    padding: 4,
+  },
+  link: {
+    textDecoration: 'none',
+    color: lightColor,
+    '&:hover': {
+      color: theme.palette.common.white,
+    },
+  },
+  button: {
+    borderColor: lightColor,
+  },
+  notifications: {
+    paddingLeft: theme.spacing(4),
+    paddingRight: theme.spacing(0),
+    marginLeft: theme.spacing(4),
+  },
+  userContainer: {
+    paddingLeft: 1,
+    display: 'flex',
+  },
+  userSpan: {
+    marginLeft: theme.spacing(1),
+  },
+  pageTitleWrapper: {
+    flexGrow: 1,
+    marginRight: 'auto',
+  },
+  pageTitle: {
+    paddingLeft: theme.spacing(2),
+    fontSize: '22px',
+    [theme.breakpoints.up('sm')]: {
+      fontSize: '26px',
+    },
+  },
+  appBarOnDrawerOpen: {
+    padding: theme.spacing(1.4),
+    zIndex: theme.zIndex.drawer+1,
+    [theme.breakpoints.between(635,732)]: {
+      padding: theme.spacing(0.75,1.4),
+    },
+    [theme.breakpoints.between(600,635)]: {
+      padding: theme.spacing(0.4,1.4),
+    },
+  },
+  appBarOnDrawerClosed: {
+    padding: theme.spacing(1.4),
+    zIndex: theme.zIndex.drawer+1,
+  },
+  toolbarOnDrawerClosed: {
+    minHeight: 59,
+    paddingLeft:24,
+    paddingRight:24,
+  },
+  toolbarOnDrawerOpen: {
+    minHeight: 58,
+    paddingLeft:20,
+    paddingRight:20,
+    [theme.breakpoints.between(620,732)]: {
+      minHeight: 68,
+      paddingLeft:20,
+      paddingRight:20,
+    },
+  },
+  itemActiveItem: {
+    color: '#4fc3f7',
+  },
+});
+
+class Header extends React.Component {
+  render() {
+    const { classes, title, onDrawerToggle ,isDrawerCollapsed} = this.props;
+    return (
+      <NoSsr>
+        <React.Fragment>
+          <AppBar color="primary" position="sticky" elevation={0} className={isDrawerCollapsed?classes.appBarOnDrawerClosed:classes.appBarOnDrawerOpen}>
+            <Toolbar className={isDrawerCollapsed?classes.toolbarOnDrawerClosed:classes.toolbarOnDrawerOpen}>

```

Comment:
Provide some spacing, to improve readability...

Reply:
Ohk Thankx for the review .!!!

Reply:
Is there any other way to complete this issue ?
I tried this way.

Reply:
I will have to check it on my local....and then confirm here.